src/backend/joanie/core/serializers/admin.py

```diff
@@ -987,3 +995,9 @@ class Meta:
     def get_total_currency(self, *args, **kwargs) -> str:
         """Return the code of currency used by the instance"""
         return settings.DEFAULT_CURRENCY
+
+    def get_owner_name(self, instance) -> str:
```

Comment:
```suggestion
    def get_owner_name(self, obj) -> str:
```

Reply:
The only reason why I'm suggesting this change is to apply the [DRF](https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield) way of using SerializerMethodField() and its method parameter

Reply:
I check in our codebase and we have no convention about that; We are using both name : obj and instance. IMO both are relevant.